### ISSUE TITLE
File missing

### ISSUE THREAD

**Issue by noart626** [BboyAkers/simple-dapp-tutorial#34]
Hi! 
nothing important but the img  https://docs.metamask.io/ethereum-metamask-chrome.png is missing for index.html ;)

**Comment by RegencySoftware** [comment]
@noart626 Are you able to run the project?  After I run... npm run serve and launch I see the 404 on the icon like you, but nothing works. Buttons do not work, no other errors on the console.

**Comment by BboyAkers** [comment]
Interesting! I'll pull it and take a look! Thanks for posting this issue 🙂

**Comment by noart626** [comment]
@RegencySoftware Yes but I make a very simple Dapps application. In fact I just used the information given in the tutorial and completed and modified the contract.js in the start repertory just to retrieve the user's token
